Software/scan_key/ErrorSQL.py

```diff
@@ -19,14 +21,12 @@ def errorSQL(id, errMessage):
 
     cursor = cnx.cursor()
 
-    os.system("hostname > tmp")
-    currMachineID = open('tmp', 'r').read()
-    currMachineID = currMachineID.split()
-    currMachineID = currMachineID[0:]
+    currMachineIDname = subprocess.check_output(['hostname'])
+    currMachineID = currMachineID.strip()
```

Comment:
Need to strip and decode the byte string before we can use it in SQL.

http://stackoverflow.com/questions/6269765/what-does-the-b-character-do-in-front-of-a-string-literal